### DISCUSSION TITLE
Anyone actually managed to complete a project?

### DISCUSSION THREAD

**Discussion by Lukemul6** [reworkd/AgentGPT#204] [Q&A]
I give it the most simple commands like "Create a very simple landing page to welcome visitors to this webpage:"
It just goes on and on adding infinite amount of tasks and can never finish. I have tried 6 different prompts all hit 25 limit

**Comment by drhouse** [comment]
ask it a coding related question, will probably work

**Comment by APiTJLillo** [comment]
Converting to a discussion, I believe I remember hearing there's going to be a discord channel for use cases.

**Comment by ligarius** [comment]
is redundant and always comes to the same thing, an unfinished task
